uber_sante/services/clinic_service.py

```diff
@@ -100,3 +100,53 @@ def modify_clinic(self, id, name, location, nb_rooms, nb_doctors, nb_nurses, ope
         self.db.write_one(update_stmt, params)
 
         return ClinicStatus.SUCCESS
+    
+    def modify_clinic_limited(self, id, location, open_time, close_time, phone):
+
+        select_stmt = '''SELECT * FROM Clinic WHERE id = ?'''
+        params = (id,)
+
+        results = self.db.read_all(select_stmt, params)
+
+        list_of_clinics = []
+
+        for result in results:
+            list_of_clinics.append(
+                Clinic(
+                    result['id'],
+                    result['name'],
+                    result['location'],
+                    result['nb_rooms'],
+                    result['nb_doctors'],
+                    result['nb_nurses'],
+                    convert_time.get_start_time_string(result['open_time']),
+                    convert_time.get_start_time_string(result['close_time']),
+                    result['phone']))
+
+        name = list_of_clinics[0].name
```

Comment:
check that the list size is at least 1 len(list_of_clinics) > 0 otherwise this will crash